etl/workspace.py

```
@@ -0,0 +1,12 @@
+"""Workspace utilities - backward compatibility module.
+
+This module provides backward compatibility for workspace utilities.
+The actual implementation is in etl.utils.workspace.
+"""
+
+# Import all workspace utilities for backward compatibility
+try:
+    from .utils.workspace import *
```

Ruff (0.12.2)

`from .utils.workspace import *` used; unable to detect undefined names

(F403)